tests/iface_namingmode/test_iface_namingmode.py

```diff
@@ -326,6 +342,12 @@ def test_show_pfc_counters(setup, setup_config_mode):
 
 class TestShowPriorityGroup():
 
+    @pytest.fixture(scope="class", autouse=True)
+    def setup_check_topo(self, duthosts, enum_rand_one_per_hwsku_frontend_hostname):
+        duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
+        if duthost.is_multi_asic:
+            pytest.skip('CLI command not supported')
```

Comment:
may be add "CLI command not supported on Multi-ASIC". Is it possible to add what command is not supported.

Reply:
I guess the CLI command can be determined from the test name.

Reply:
Yes, the testcase name will give the failed CLI, so not adding the CLI command in the message